main.cpp

```diff
@@ -188,98 +129,127 @@ int main()
 
     }
 
-    /*********************************************************************************
-    ** This area deletes the dynamically allocated Critter array
-    *********************************************************************************/
-
-    for(int i = 0; i <  size_y; i++)
+/*********************************************************************************
+** This area deletes the dynamically allocated Critter array
+*********************************************************************************/
+    for(int i = 0; i <  size_x; i++)
     {
-      for(int j = 0; j < size_x; j++)
```

Comment:
switching x and y around doesn't cause any bugs?

Reply:
No because I was consistent with where it got changed.  I wanted it to be consistent for how we were using rows and columns within the program.